app.ts

```diff
@@ -3,7 +3,7 @@ const express = require('express');
 require('express-async-errors');
 const cors = require('cors');
 const TokenValidator = require('twilio-flex-token-validator').validator;
-const crypto = require('crypto');
+const cryptoLib = require('crypto');
```

Comment:
Got some naming conflict warnings in WebStorm which is why I renamed some of these variables